lib/web/ui/app_test.go

```diff
@@ -88,11 +86,8 @@ func TestMakeApps(t *testing.T) {
 			expected: []App{},
 		},
 		{
-			name: "app without user groups",
-			apps: types.Apps{
-				newApp(t, "1", "1.com", "", map[string]string{"label1": "value1"}),
-				newApp(t, "2", "2.com", "group 2 friendly name", map[string]string{"label2": "value2", types.OriginLabel: types.OriginOkta}),
-			},
+			name:      "app without user groups",
+			appsOrSPs: types.AppServersOrSAMLIdPServiceProviders{createAppServerOrSPFromApp(newApp(t, "1", "1.com", "", map[string]string{"label1": "value1"})), createAppServerOrSPFromApp(newApp(t, "2", "2.com", "group 2 friendly name", map[string]string{"label2": "value2", types.OriginLabel: types.OriginOkta}))},
```

Comment:
you must like have a very wide monitor or something lolol, lets break this (and below)